projects/demo/src/modules/directives/skeleton/examples/2/index.html

```diff
@@ -0,0 +1,24 @@
+<label tuiLabel>
+    <input
+        tuiToggle
+        type="checkbox"
+        [(ngModel)]="skeleton"
+    />
+    Show skeleton
+</label>
+
+<p>
+    <span
+        [tuiSkeleton]="
+            skeleton
+                ? 'This text serves as the content behind the skeleton and depending on its length, the skeleton will adjust to fit it.'
+                : ''
+        "
+    >
+        {{ skeleton ? '' : 'This text will be replaced by a placeholder.' }}
+    </span>
+</p>
```

Comment:
Is this additional nesting is really required ?

Could we simplify example ?
```suggestion
<p
    [tuiSkeleton]="
            skeleton
                ? 'This text serves as the content behind the skeleton and depending on its length, the skeleton will adjust to fit it.'
                : ''
        "
>
    {{ skeleton ? '' : 'This text will be replaced by a placeholder.' }}
</p>
```